aws/resource_aws_ebs_volume.go

```diff
@@ -26,7 +27,10 @@ func resourceAwsEbsVolume() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 
-		CustomizeDiff: resourceAwsEbsVolumeCustomizeDiff,
+		CustomizeDiff: customdiff.All(
```

Comment:
similar comment in the `e* 2` PR , `customdiff.Sequence` _may_ be preferred here